autoprotocol/container_type.py

```diff
@@ -6,7 +6,7 @@
     :license: BSD, see LICENSE for more details
 
 """
-
+# fmt: off
```

Comment:
why are you turning off the Black formatter here? Don't like it? 🙈

Reply:
Oh whoops, I'll fix that haha